### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by eliasmalik**
Related to #4 

### Changes
* Copy over config files, starter docs and static server from `twine-visitor`
* Add some standard dependencies

***Note***:
I think this particular PR should not be squashed, b/c the commits are self contained and will themselves be more descriptive/accurate than a single squashed commit.

### CODE REVIEW

File: docs/architecture.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+### Root
+
+```
+./
+|- bin          Developer and deployment scripts
+|- docs         Documentation
+|- public       Public assets
+|- src          Client app
+|  |- api             Interface with the Twine HTTP API
+|  |- shared          All code shared between the admin and visitor UIs
```
Comment (astroash) [inline docs/architecture.md]: 🔥

File: docs/config_guide.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+REACT_APP_API_HOST_DOMAIN=<base_url>
+```
+
+## Server
+The server may be configured with the following environment variables:
+```sh
+# Hostname from which to redirect
+# MUST NOT INCLUDE PROTOCOL
+# Example:
+#   twine-visitor.herokuapp.com
```
Comment (astroash) [inline docs/config_guide.md]: twine-visitor.herokuapp.com -> twine-data.herokuapp.com

File: docs/stack.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Stack
+
+## Contents
+1. [React](#react)
+1. [React Router](#react-router)
+1. [Styled Components](#styled-components)
+1. [Instascan](#instascan)
```
Comment (astroash) [inline docs/stack.md]: 🔥

**Review by astroash — CHANGES_REQUESTED** [review]

File: docs/architecture.md (hunk 1 above)
Comment (eliasmalik) [reply docs/architecture.md]: We still need a folder to put common things in. You don't like the way `shared` is structured in the visitor app currently, that's fine, but that stuff still needs to go somewhere.

Comment (astroash) [reply docs/architecture.md]: Can you give me an example of some thing that is in our visitor app shared folder that will not fit into another directory.

Comment (astroash) [reply docs/architecture.md]: Can you give me an example of some thing that is in our visitor app shared folder that will not fit into another directory.

Comment (eliasmalik) [reply docs/architecture.md]: All of the common form components, the nav header. Probably the constants.js file too.

Comment (astroash) [reply docs/architecture.md]: `shared` previously made sense as we were splitting between two different sections of the app, the `visitor` view and the `admin` view. I do not think that we need this style of sharing anymore, and any common code can be found under their own directories.

Comment (eliasmalik) [reply docs/architecture.md]: What directories? Where do we keep components that are used on several different parts of the app?

Comment (astroash) [reply docs/architecture.md]: Can we have a components folder?

Comment (astroash) [reply docs/architecture.md]: Can we have a components folder?

**Review by astroash — APPROVED** [review]

**Pull request merged**